microsoft-edge/webview2/how-to/runtime-channel.md

```diff
@@ -0,0 +1,188 @@
+---
+description: How to specify a WebView2 Runtime channel for your app, to test experimental APIs in a prerelease package.
+title: Specify a WebView2 Runtime channel for your app
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 07/14/2021
+ms.topic: how-to
+ms.prod: microsoft-edge
+ms.technology: webview
+keywords: IWebView2, IWebView2WebView, webview2, webview, win32 apps, win32, edge, ICoreWebView2, ICoreWebView2Host, browser control, edge html
+---
+# Specify a WebView2 Runtime channel for your app
+
+<!-- is the phrase "runtime channel" correct?  It doesn't appear in the docs repo.  "WebView2 Runtime" appears 102/14, and "Microsoft Edge channel" appears 13/5. -->
+
+To develop a WebView2 application, you need the WebView2 SDK and a WebView2 Runtime.  The WebView2 SDK contains API signatures, which allow you to write code using WebView2 APIs in your app.  But to see your WebView2 code build and run, you need a runtime channel,<!--change to "Microsoft Edge channel"? --> which contains the Microsoft Edge binaries that are needed to render your app.
```

Comment:
```suggestion
To develop a WebView2 application, you need the WebView2 SDK and a WebView2 Runtime / Browser Channel.  The WebView2 SDK contains API signatures, which allow you to write code using WebView2 APIs in your app.  But to see your WebView2 code build and run, you need a runtime channel,<!--change to "Microsoft Edge channel"? --> which contains the Microsoft Edge binaries that are needed to render your app.
```

Reply:
Addressing your comment: ,'change to "Microsoft Edge channel"'

The developer will be able to use the webview2 runtime OR browser channel. My wording 'runtime channel' was an attempt to capture both. Perhaps we can specify both if you'd like?

Reply:
I researched existing wording in the repo, and came up with full & abbreviated phrases:
"a Microsoft Edge preview channel or the WebView2 Runtime"
"a channel or the Runtime"
"channel or Runtime"